src/mmw/js/src/compare/views.js

```diff
@@ -1008,10 +1090,85 @@ function getTr55Tabs(scenarios) {
     ]);
 }
 
+function mapScenariosToHydrologyChartData(scenarios, key) {
+    return scenarios
+        .map(function(scenario) {
+            return scenario
+                .get('results')
+                .models
+                .reduce(function(accumulator, next) {
+                    if (next.get('displayName') !== models.constants.HYDROLOGY) {
+                        return accumulator;
+                    }
+
+                    return accumulator.concat(next
+                        .get('result')
+                        .monthly
```

Comment:
It seemed like there would be a `.get('monthly')` way to get this attribute but for whatever reason that didn't work.